src/server/wtr-config.js

```diff
@@ -95,6 +95,9 @@ export class WTRConfig {
 								line-height: 1.4rem;
 								padding: 30px;
 							}
+							body[data-theme="dark"] {
+								background-color: #000000;
```

Comment:
As I think about this more, I think we already have this risk with standard light theme tests. If someone removed the background color, the test would likely still pass because the default `#ffffff` is the same.

Reply:
I'm OK just deferring and not adding built-in support for this in the vdiff library until we're more confident in our approach. It would be easy for core to handle this itself like you said.